manager/system_handler.py

```diff
@@ -185,6 +185,7 @@ def get_systems(self):
         response = {}
         response['data'] = self._format_data(list_arguments["data_format"], data)
         response['meta'] = system_view.get_metadata()
+        response['meta']['opt_out'] = bool(args['opt_out']) if args['opt_out'] is not None else ''
```

Comment:
I think it would be better if SystemView just overrides get_metadata(), send-super, and add opt_out to the result

Reply:
then the same needs to be done in `vulnerabilities_handler.py`

Reply:
Yeah, I see what you mean. I think the problem is that we have a confusion of ownership - get_metadata() is owned by the *View, but the *Handler knows what/how to add to it. This is ugly, but I don't think we can do the refactoring it'll take right now.

I withdraw the comment, will continue testing.